docs/usage/00-installing.adoc

```diff
@@ -37,11 +37,12 @@ To install `dklb`, the following commands may be run from the root of this repos
 
 [source,console]
 ----
+$ hack/service-account.sh
 $ kubectl create -f docs/deployment/00-prereqs.yaml
 $ kubectl create -f docs/deployment/10-deployment.yaml
 ----
 
-This will create a `dklb` deployment in the `kube-system` namespace having two replicas:
+This will create DC/OS service account for dklb, a Kubernetes secret named `dklb-dcos-config` and `dklb` deployment in the `kube-system` namespace having two replicas:
```

Comment:
```suggestion
This will create a DC/OS service account for dklb, a Kubernetes secret named `dklb-dcos-config` and `dklb` deployment in the `kube-system` namespace having two replicas:
```